src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -937,6 +937,73 @@ def test_appconfig_to_appservice_import_export(self, resource_group, location):
         slot = self.create_random_name(prefix='Slot', length=24)
         self.cmd('webapp deployment slot create -g {} -n {} -s {}'.format(resource_group, webapp_name, slot))
 
+        # App configuration reference tests
+        # Create new key-values
+        entry_key = 'TestKey'
+        entry_value = 'TestValue'
+        entry_label = 'AppServiceReferenceExport'
```

Comment:
Do you mind adding a test for null label too? Just to make sure that the reference string is formed as expected.